data/Makefile

```diff
@@ -9,10 +10,10 @@ env | grep ECOBALYSE_DATA_DIR || echo "No ECOBALYSE_DATA_DIR in environment. Con
 env | grep ECOBALYSE_DATA_DIR || exit
 @if [ "$(shell docker container inspect -f '{{.State.Running}}' $(NAME) )" = "true" ]; then \
   echo "(Using the existing container)" &&\
-	docker exec -u jovyan -it -e ECOBALYSE_DATA_DIR=/home/jovyan/ecobalyse-private/ -e PYTHONPATH=. -w /home/jovyan/ecobalyse/data $(NAME) $(1);\
+	docker exec -u jovyan -it -e ECOBALYSE_DATA_DIR=/home/jovyan/ecobalyse/public/ -e PYTHONPATH=. -w /home/jovyan/ecobalyse/data $(NAME) $(1);\
 else \
 	echo "(Creating a new container)" &&\
-  docker run --rm -it -v $(NAME):/home/jovyan -v $$PWD/../:/home/jovyan/ecobalyse -v $$PWD/../../dbfiles/:/home/jovyan/dbfiles -v $(ECOBALYSE_DATA_DIR):/home/jovyan/ecobalyse-private -e ECOBALYSE_DATA_DIR=/home/jovyan/ecobalyse-private/ -w /home/jovyan/ecobalyse/data $(NAME) $(1); fi
+  docker run --rm -it -v $(NAME):/home/jovyan -v $$PWD/../:/home/jovyan/ecobalyse -v $$PWD/../../dbfiles/:/home/jovyan/dbfiles -v $(ECOBALYSE_DATA_DIR):/home/jovyan/ecobalyse-private -e ECOBALYSE_DATA_DIR=/home/jovyan/ecobalyse/public/ -w /home/jovyan/ecobalyse/data $(NAME) $(1); fi
```

Comment:
How about spreading long lines like this one to improve legibility:

```
docker run --rm -it \
  -v $(NAME):/home/jovyan \
  -v $$PWD/../:/home/jovyan/ecobalyse \
  -v $$PWD/../../dbfiles/:/home/jovyan/dbfiles \
  -v $(ECOBALYSE_DATA_DIR):/home/jovyan/ecobalyse-private \
  -e ECOBALYSE_DATA_DIR=/home/jovyan/ecobalyse/public/ \
  -w /home/jovyan/ecobalyse/data \
  $(NAME) $(1); fi
```

Please note that there's still a reference to `ecobalyse-private` btw.